bin/release.sh

```diff
@@ -71,6 +73,17 @@ update_version() {
     # check the current version of the crate
     local curr_version
     curr_version=$(cargo pkgid -p "$crate" | sed -n 's/.*#\(.*\)/\1/p')
+
+    if ! cargo --list | grep -q "set-version"; then
+        err "missing cargo-set-version executable (from cargo-edit)"
+        if confirm "       install it?"; then
+            cargo install cargo-edit
+        else
+            echo "okay, exiting"
+            exit 1
+        fi
+    fi
```

Comment:
what's the motivation for introducing an additional dep for doing this when we were previously able to do it with a single line of `sed`? i'm open to the change but would like to understand what this gets us that the sed script doesn't.

Reply:
I used `cargo ugrade` from `cargo-edit` to set the version of `console-api` on `console-subscriber` and `tokio-console`. I could probably have done it with a `sed` script too, but it's a bit harder (and more difficult to maintain).

Since I was installing `cargo-edit` for that anyway, it made sense to use it here too (again, I feel it's easier to maintain).

Using `sed` also has the downside of not always being compatible of different unixes (*cough* BSD/macOS *cough*).

Reply:
cool, makes sense to me! 👍